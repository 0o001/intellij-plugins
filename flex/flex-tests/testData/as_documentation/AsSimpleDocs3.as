package controls
{

/**
 *  The CCC control is super special control.
 *  <ul>
 *    <li>Which CSS style properties the class inherits</li>
 *    <li>Which style properties are colors, and therefore get special handling</li>
 *    <li>A list of strings that are aliases for color values</li>
 *  </ul>
 * @see controls.CCC
 * @see controls.CCC
 */
public interface CCC {
    function aaa():C<caret>CC
  }
}
