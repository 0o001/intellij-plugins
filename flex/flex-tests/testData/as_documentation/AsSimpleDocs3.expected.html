  The CCC control is super special control.<BR>
  <ul><BR>
    <li>Which CSS style properties the class inherits</li><BR>
    <li>Which style properties are colors, and therefore get special handling</li><BR>
    <li>A list of strings that are aliases for color values</li><BR>
  </ul><DL><DT><b>See also:</b></DT><DD><a href="psi_element://controls.CCC">controls.CCC</a></DD><DD><a href="psi_element://controls.CCC">controls.CCC</a></DD></DL>